acorn/index.d.ts

```diff
@@ -228,19 +218,23 @@ declare namespace acorn {
 
     const version: string;
 
-    function parse(input: string, options?: Options): ESTree.Program;
+    interface IParse {
+        (input: string, options?: Options): ESTree.Program;
+    }
+
+    const parse: IParse;
 
     function parseExpressionAt(input: string, pos: number, options?: Options): ESTree.Expression;
 
     function tokenizer(): Parser;
 
-    let parse_dammit: Function;
+    let parse_dammit: IParse;
     let LooseParser: LooseParser;
     let pluginsLoose: {};
```

Comment:
While you're at it, this (and `plugins` below) should not be types of empty objects, but types of extensible objects with proper dynamic key-value type definition. Otherwise contents of these collections are inaccessible without extra cast.